### DISCUSSION TITLE
How to update Parcelvoy in Docker?

### DISCUSSION THREAD

**Discussion by ourcaldo** [parcelvoy/platform#459] [Q&A]
Hi everyone,

I've successfully installed Parcelvoy on Ubuntu using Google Cloud Platform. However, I'm unsure about updating Parcelvoy from Github. How can I do this?

Thank you.

**Comment by pushchris** [comment]
If you are using the Docker Compose file from the repo it uses the latest image by default, to pull whatever the latest is you just need to run `docker-compose pull` and then restart `docker-compose up -d`. Alternatively you can pin what version to use by setting the `PARCELVOY_VERSION` environment variable

**Reply by ourcaldo** [reply]
Hi @pushchris thank you for your answer,

So, as long as I use the latest docker-compose.yml it always pulls the latest version from the Parcelvoy main repository with all commits that have changed or added?

Thanks

**Reply by pushchris** [reply]
You'll still need to tell Docker Compose to pull the most recent image tagged with `latest` by running `docker-compose pull`.

This does not re-create the Docker image from code however if that is what you are asking, this just looks at the releases and loads the image of the most recent release (which is what I would highly recommend doing since there is no guarantee that merged code is ready for use until a release). If you are wanting to manually create a Docker image yourself you will need to run `npm run docker:build` and then publish that image yourself somewhere so that you can pull it down in GCP.
